test/LinalgExt/invalid.mlir

```diff
@@ -0,0 +1,44 @@
+// RUN: mlir-opt %s -split-input-file -verify-diagnostics
+
+// -----
+
+func @not_enough_results() -> () {
+  %num_threads = arith.constant 100 : index
+  // expected-error@+1 {{'linalg_ext.in_parallel' op has 1 results, but its terminator produces 0 values}}
```

Comment:
`s/produces/has` of `s/produces/yields` here, below and in the .cpp ?

I'd expect to see a value returned if it were produced but here the terminator has no results.

Reply:
Done!